core/src/jsMain/kotlin/dev/fritz2/dom/html/elements.kt

```diff
@@ -1337,7 +1404,7 @@ interface TagContext : WithJob, WithScope {
      */
     fun <T> Flow<T>.asString(): Flow<String> = this.map { it.toString() }
```

Comment:
Do we wanna keep that helper function? In my opinion this is confusion cause it appears next to `asText()` and is only a simple mapping instead of creating a mount-point.